### DISCUSSION TITLE
could import contacts dialog be misleading?

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#245] [General]
Hi,
If you import contacts on Fossify Contacts

1. Settings
2. Import contacts from vcf
3. select the file to import

Now a dialog appears to select in which address book you want the contacts to be imported.

It states:

title: Import contacts
field's title: target contact **source**

But source means where something comes from, the origin, and not the destination. So it seems that it is asking where the contact was.
Isn't it misleading?

![Imagepipe_0](https://github.com/user-attachments/assets/10bad4a0-71b8-450e-9815-f1a85e1300a9)

**Comment by Aga-C** [comment]
Please create a feature request in [Contacts](https://github.com/FossifyOrg/Contacts/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) with a suggestion how it should be written.
